src/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.SecKeyRef.cs

```diff
@@ -357,5 +357,18 @@ namespace System.Security.Cryptography.Apple
 {
     internal sealed class SafeSecKeyRefHandle : SafeKeychainItemHandle
     {
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing && SafeHandleCache<SafeSecKeyRefHandle>.IsCachedInvalidHandle(this))
```

Comment:
should this be `||`?

Reply:
No? :smile: If we're finalizing the cached handle, let it finalize.  If we're disposing and it's the cached handle, ignore the call to dispose.  If we're disposing and it's not the cached handle, continue disposing.